### ISSUE TITLE
New Antline Router Items

### ISSUE THREAD

**Issue by libbybapa** [BEEmod/BEE2-items#1445]
The antline router and router strip are both very useful.  

I have also gotten sneaky enough to use the signage item with the timer set to infinite and the arrow off as an additional invisible router item...  

Another useful router would be a 1/2X1/2 router square that could be placed in the 8 positions of the 1/2X1/2X1/2 block.

One other could be the centered 1/2X1/2 square of a square light, but invisible.

**Comment by LautaroL20** [comment]
A laser relay in an Antline Router format could work. It would have the 5 dot positions.

**Comment by Luke18033** [comment]
@LautaroL20 - 5-dot positioning only works with the laser items.

**Comment by libbybapa** [comment]
Right.  A laser relay that is invisible in the compiled map...voila new antline router object.

**Comment by HugoBDesigner** [comment]
Previous versions of the Signage item behaved like a 2x1 antline router (you could disable the arrow and set the signage to infinite). I believe it'd be trivial to make this one, given that a similar item already existed (before it was changed from the rotating handle to the 8 position one).
